manage/config-dialog.js

```diff
@@ -297,6 +314,10 @@ function configDialog(style) {
     }
   }
 
+  function focusInput(event) {
```

Comment:
This is a misleading function name: it doesn't focus the input. Maybe onInputFocused or selectAllOnFocus to match the existing naming scheme